data/ships.txt

```diff
@@ -22,7 +22,6 @@ ship "SpaceJumper"
 		"LP036a Battery Pack"
 		
 	engine 0 19
-		under
 	explode "tiny explosion" 10
 	description `The StarGate Inc. SpaceJumper is the smallest craft capable of fitting a hyperdrive. It's sold in kit form and advertised to "everyone who ever wanted their own spaceship". Most buyers who manage to assemble the ship make one terrifying trip to Alpha Centauri and back, and then sell the thing for scrap.`
```

Comment:
```suggestion
	description `The StarGate Inc. Space Jumper is the smallest craft capable of fitting a hyperdrive. It's sold in kit form and advertised to "everyone who ever wanted their own spaceship". Most buyers who manage to assemble the ship make one terrifying trip to Alpha Centauri and back, and then sell the thing for scrap.`
```

Reply:
And about the engine: I’m pretty sure they are under by default as well.

Reply:
Actually, it would appear I’m wrong according to [line 2613 in Engine.cpp](https://github.com/endless-sky/endless-sky/blob/51de725e6eac78abf413e8025cc7e8efc3c91c7b/source/Engine.cpp#L2613), right?
The question is, should it be under? Based on all the other ships no (the only use of `under` in `human/ships.txt` is for fighter and drone bays), but then again it’s your choice.

Reply:
I eyeballed it and without under looked fine. The smooshed together "SpaceJumper" instead of "Space Jumper" is a deliberate obnoxiousness. All other suggested changes look good.